File: frontend/lib/theme/seed_colors.dart
import 'package:flutter/material.dart';

/// One of the six predefined seed colors users can pick from in the theme
/// settings screen. Chosen to evoke the app's domain (Proton/Wine/Steam on
/// Linux) rather than generic swatches.
class SeedColorOption {
  const SeedColorOption({required this.name, required this.color});

  final String name;
  final Color color;
}

const List<SeedColorOption> kSeedColorOptions = <SeedColorOption>[
  SeedColorOption(name: 'Proton Violet', color: Color(0xFF7C4DFF)),
  SeedColorOption(name: 'Steam Blue', color: Color(0xFF2A6DF4)),
  SeedColorOption(name: 'GE Ember', color: Color(0xFFFF6D3F)),
  SeedColorOption(name: 'Wine Crimson', color: Color(0xFFB3261E)),
  SeedColorOption(name: 'Terminal Green', color: Color(0xFF00C853)),
  SeedColorOption(name: 'Slate Teal', color: Color(0xFF00696D)),
];

File: frontend-compose/src/main/kotlin/com/protonctl/app/theme/SeedColors.kt
package com.protonctl.app.theme

import androidx.compose.ui.graphics.Color

data class SeedColorOption(val displayName: String, val color: Color)

val SeedColorOptions = listOf(
    SeedColorOption("Proton Violet", Color(0xFF7C4DFF)),
    SeedColorOption("Steam Blue", Color(0xFF2A6DF4)),
    SeedColorOption("GE Ember", Color(0xFFFF6D3F)),
    SeedColorOption("Wine Crimson", Color(0xFFB3261E)),
    SeedColorOption("Terminal Green", Color(0xFF00C853)),
    SeedColorOption("Slate Teal", Color(0xFF00696D)),
)
